apps/ledger-live-mobile/src/screens/BleDevicePairingFlow/index.tsx

```diff
@@ -70,7 +70,7 @@ export const BleDevicePairingFlow: React.FC<Props> = ({ navigation }) => {
       <Flex px={6} flex={1}>
         <BleDevicePairingFlowComponent
           key={keyToReset}
-          filterByDeviceModelId={DeviceModelId.stax}
+          filterByDeviceModelId={QRCodeDevices}
```

Comment:
could be clearer with `[DeviceModelId.stax, DeviceModelId.europa]` here

Reply:
It's a list that contains all the devices with screen and could be (surely) extensible in the future, so let's maintain the list to avoid modifying everywhere when a new device comes out.

Reply:
This list could have a better name then, it's not clear to have `QrCodeDevices` filtered for `BlePairingFlow` imho